tests/common/gcp_api/container_test.py

```diff
@@ -45,17 +45,45 @@ def test_no_quota(self, mock_google_credential):
         container_client = container.ContainerClient(global_configs={})
         self.assertEqual(None, container_client.repository._rate_limiter)
 
-    def test_get_serverconfig(self):
-        """Test get KE serverconfig."""
+    def test_get_serverconfig_zone(self):
```

Comment:
nit: do you think that this test case would be nicer to read if the method name has a`by`
```
def test_get_serverconfig_by_zone(self):
```